server/controllers/users.js

```diff
@@ -34,14 +34,17 @@ export const exists = async (req, res) => {
 export const signin = (req, res, next) => {
   const { user, redirect, websiteUrl } = req.body;
   let loginLink;
+  let clientIP;
   return models.User.findOne({ where: { email: user.email.toLowerCase() } })
     .then(u => u || models.User.createUserWithCollective(user))
     .then(u => {
       loginLink = u.generateLoginLink(redirect || '/', websiteUrl);
+      clientIP = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
```

Comment:
Should be `req.ip` https://expressjs.com/en/api.html#req.ip

We expect it to be accurate, because we have it properly configured in express. See: https://github.com/opencollective/opencollective-api/blob/master/server/lib/express.js#L26